src/components/Header/index.tsx

```diff
@@ -1,34 +1,29 @@
-import { useState, useEffect } from "react";
+import { useEffect } from "react";
 import { useMediaQuery } from "@mui/material";
-import HeaderDesktop from "./HeaderDesktop";
-import HeaderTabletAndMobile from "./HeaderTabletAndMobile";
-import USABanner from "./USABanner";
-import GenericAlert from "../GenericAlert";
+import { useSnackbar } from "notistack";
+import HeaderDesktop from "./components/HeaderDesktop";
+import HeaderTabletAndMobile from "./components/HeaderTabletAndMobile";
+import USABanner from "./components/USABanner";
 import { useAuthContext } from "../Contexts/AuthContext";
 
 const Header = () => {
+  const { enqueueSnackbar } = useSnackbar();
+  const { error: authError } = useAuthContext();
   const tabletAndMobile = useMediaQuery("(max-width: 1024px)");
 
-  const [showLoginError, setShowLoginError] = useState<boolean>(false);
-  const authContext = useAuthContext();
-
   useEffect(() => {
-    if (authContext.error !== undefined) {
-      setShowLoginError(true);
-      setTimeout(() => setShowLoginError(false), 10000);
+    if (typeof authError !== "string") {
```

Comment:
Instead of returning here, should we instead display a generic error message, like an unknown authentication error has occurred or something like that? Since authError is still populating, it is still an error.

Reply:
Agreed. Thanks for catching this issue. 62a503e